src/condor_includes/condor_crypt.h

```diff
@@ -26,6 +26,27 @@
 
 #include "CryptKey.h"
 
+struct StreamCryptoState {
+    // The IV is split in two: a 32-bit counter and a
+    // 64-bit random number for a total of 12 bytes.
+    // The ctr is added to the last 4 bytes of the IV.
+    union Packed_IV {
+        unsigned char iv[16]{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+        struct {
+            uint32_t pkt;
```

Comment:
Drop the connection counter; switch `pkt` to 64-bit to avoid theoretical overflows.

Reply:
As discussed, keeping it at 32 bit... still plenty big, and we check to see if the counter reaches 0xffffffff.